src/migration-wizard/steps/discovery/EnhancedDownloadButton.tsx

```
@@ -0,0 +1,654 @@
+        return `${(num / 1000).toFixed(1)}K`;
+      }
+      return num.toString();
+    };
+
+    const generateOSTable = (): string => {
+      const osEntries = Object.entries(vms.os).sort(([,a], [,b]) => (b as number) - (a as number));
+      if (osEntries.length === 0) {
+        return '<tr><td colspan="4">No operating system data available</td></tr>';
+      }
+
+      return osEntries.map(([osName, count]) => {
```

ESLint

[error] Replace `······return·osEntries` with `········return·osEntries⏎··········`

(prettier/prettier)